src/pages/index.js

```diff
@@ -18,28 +20,72 @@ const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+const profileImageContainer = document.querySelector(
+  ".profile__image-container"
+);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "fd75a5c6-0c34-4591-90ec-6f1fc1e0d0b3",
+    "Content-Type": "application/json",
+  },
+});
 
 const addCardFormValidator = new FormValidator(configItems, addCardFormElement);
 const profileEditValidator = new FormValidator(configItems, profileEditForm);
+const editAvatarFormValidator = new FormValidator(
+  configItems,
+  document.querySelector("#avatar-edit-form")
+);
 
 addCardFormValidator.enableValidation();
 profileEditValidator.enableValidation();
+editAvatarFormValidator.enableValidation();
 
 const cardList = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (item) => {
       const cardElement = createCard(item);
       cardList.addItem(cardElement);
     },
   },
   ".cards__list"
 );
-cardList.renderItems();
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    console.log("Cards from API:", cards);
+    if (Array.isArray(cards)) {
+      cards.forEach((card) => {
+        const cardElement = createCard(card);
+        cardList.addItem(cardElement);
```

Comment:
Please, use `renderItems`  for rendering the array of cards


You can change it to make it accept the array into the call


```js
  .then((cards) => {
    if (Array.isArray(cards)) {
      cardList.renderItems(cards);
    }
  })

```